projects/admin/src/app/routes/route-tool.service.ts

```diff
@@ -190,4 +235,14 @@ export class RouteToolService {
     }
     return null;
   }
+
+  /**
+   * can
+   * @param can - boolean
+   * @param message - string
+   * @return Observable
+   */
+  private canObservable(can: boolean, message: string = '') {
```

Comment:
_ is missing for private property, but this method seems a bit overkill as it contains only one line. We can also consider to use decorators: https://www.typescriptlang.org/docs/handbook/decorators.html

Reply:
remove function